src/index.tsx

```diff
@@ -89,8 +89,6 @@ if (environment.sentryDSN) {
   });
 }
 
-const TranslatedApp = withTranslation()(App);
```

Comment:
This line was superfluous in our implementation and was therefore removed. The approach of using `useTranslation` within the components that need translation provides more flexibility to the developer.

https://react.i18next.com/latest/withtranslation-hoc